include/swift/SIL/DebugUtils.h

```diff
@@ -49,9 +49,10 @@ inline bool isDebugInst(SILInstruction *Inst) {
   return isa<DebugValueInst>(Inst) || isa<DebugValueAddrInst>(Inst);
 }
 
-/// Returns true if the instruction \p Inst is a maintenance instructions which
-/// is relevant for debug information and does not get lowered to an instruction.
-inline bool isMaintenanceInst(SILInstruction *Inst) {
+/// Returns true if the instruction \p Inst is a meta instruction which
+/// is relevant for debug information and does not get lowered to a real
+/// instruction.
+inline bool isMetaInstruction(SILInstruction *Inst) {
```

Comment:
Can you make this a method of SILInstruction?
I also believe that every SILInstruction that implements getVarInfo() should be included in this list (DebugValueAddr, AllocBox, ...) .